assets/src/js/admin/reports/components/donor-item/utils/index.js

```diff
@@ -11,6 +11,7 @@ export function getBGColor() {
 	return palette[ Math.floor( Math.random() * ( palette.length ) ) ];
 }
 
+
```

Comment:
Let's clean this up so this file isn't included.

Reply:
@JasonTheAdams closing this because this file is not part of this pr.